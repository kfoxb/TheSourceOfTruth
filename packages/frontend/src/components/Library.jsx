import React, { Fragment } from 'react';
import styled from 'styled-components';
import { PUBLISHED } from '@the-source-of-truth/shared/constants';
import DocumentsContainer from '../containers/DocumentsContainer';

const StyledDiv = styled.div`
  display: grid;
  grid-template-columns: repeat(auto-fit, 250px);
  grid-gap: 20px;
  justify-content: center;
`;

export default function Library() {
  return (
    <Fragment>
      <h3>Library</h3>
      <StyledDiv>
        <DocumentsContainer phase={PUBLISHED} />
      </StyledDiv>
    </Fragment>
  );
}
