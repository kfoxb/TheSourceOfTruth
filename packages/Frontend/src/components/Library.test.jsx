/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import Library from './Library';

describe('Library', () => {
  it('should render', () => {
    expect(shallow(<Library />)).toMatchSnapshot();
  });
});
