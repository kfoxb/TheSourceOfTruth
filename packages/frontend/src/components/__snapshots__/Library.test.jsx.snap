// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Library should render 1`] = `
<React.Fragment>
  <h3>
    Library
  </h3>
  <styled.div>
    <Connect(DocumentsContainer)
      phase="published"
    />
  </styled.div>
</React.Fragment>
`;
